runner/local/simple.go

```diff
@@ -198,7 +197,24 @@ func (r *simpleRunner) run(cmd *runner.Command, runId runner.RunId, doneCh chan
 
 	switch st.State {
 	case execer.COMPLETE:
-		r.updateStatus(runner.CompleteStatus(runId, st.ExitCode))
+		snapshotId := ""
+		srcToDest := map[string]string{
```

Comment:
(Feel free to ignore; happy to do this in another PR)

Here's a way you could accomplish this without IngestMap (in pseudocode; excuse typos/wrong method names):

outputSnapDir := tempDir.NewTempDir()
ioutil.Cp(stdout.AsFile(), path.Join(outputSnapDir, "stdout"))
ioutil.Cp(stderr.AsFile(), path.Join(outputSnapDir, "stderr"))

snapID := r.filer.Ingest(outputSnapDir)